OpenRA.Mods.Common/Traits/World/MusicPlaylist.cs

```diff
@@ -52,13 +53,26 @@ public class MusicPlaylist : INotifyActorDisposing, IGameOver
 		public MusicPlaylist(World world, MusicPlaylistInfo info)
 		{
 			this.info = info;
-			this.world = world;
 
-			IsMusicInstalled = world.Map.Rules.InstalledMusic.Any();
+			// Build InstalledMusic preferring existing in Map.Music over Rules.InstalledMusic
```

Comment:
Convention is to do this in `Ruleset.Load`, where we are currently explicitly passing in an empty set of map rules to merge in.